cmd/log-errors.go

```diff
@@ -0,0 +1,1061 @@
+package cmd
```

Comment:
Add license header, we should move the logger to a new package along with errorIf and fatalIf.